package.json

```diff
@@ -20,7 +20,7 @@
     "format:yaml": "prettier --write --parser yaml '**/*.yaml'",
     "format:graphql": "prettier --write --parser graphql '**/*.graphql'",
     "start": "yarn playground:start",
-    "prerelease": "yarn build && yarn typecheck",
+    "prerelease": "npm_config_registry=https://registry.npmjs.org npm whoami && yarn build && yarn typecheck",
```

Comment:
`preprerelease` would work I think. A joke but still.

Could also be `"auth": npm_config_registry=https://registry.npmjs.org npm whoami `

Reply:
`auth` or `auth:npm` is a good idea I guess